packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -245,6 +245,65 @@ any attempt to save more than one will result in an error.
 You can use the [`list-source-credentials` AWS CLI operation](https://docs.aws.amazon.com/cli/latest/reference/codebuild/list-source-credentials.html)
 to inspect what credentials are stored in your account.
 
+## Test reports
+
+You can specify a test report in your buildspec:
+
+```typescript
+const project = new codebuild.Project(this, 'Project', {
+  buildSpec: codebuild.BuildSpec.fromObject({
+    // ...
+    reports: {
+      myReport: {
+        files: '**/*',
+        'base-directory': 'build/test-results',
+      },
+    },
+  }),
+});
+```
+
+This will create a new test report group,
+with the name `<ProjectName>-myReport`.
+Since the project's role will need additional permissions to create that resource,
+you have the the `createsReportGroups()` method:
+
+```typescript
+project.createsReportGroups();
```

Comment:
Maybe turn this into `ReportGroup.grantCreate(project)` or something, to follow the typical naming pattern & make it clear what this does?

Reply:
Like we talked about, got rid of this completely, in favor of always adding these permissions.